docs/astro/sql/operators/export.rst

```diff
@@ -0,0 +1,30 @@
+======================================
+export_file operator
+======================================
+
+.. _export_file_operator:
+
+When to use the ``export_file`` operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The ``export_file`` function writes SQL table to csv/parquet on local/S3/GCS.
+
+There are two use cases of the ``export_file`` operator.
+
+#. Export file from table.
+#. Export file from dataframe.
+
+Case 1: Export file from table.
```

Comment:
@sunank200 I think we should also explain from what location we can import from.

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/633/commits/58f407bd813492247e2bc4bc31c010df06a072df)

Reply:
@sunank200 I this should be listed referring to Supported database section.